elixir/apps/web/test/web/live/settings/identity_providers/microsoft_entra/connect_test.exs

```diff
@@ -0,0 +1,317 @@
+defmodule Web.Live.Settings.IdentityProviders.MicrosoftEntra.Connect do
```

Comment:
On the next provider probably we should move parts of the implementations to the adapter and unify all those `.Connect` modules, because they contain minimal changes while we duplicate large sets of tests.

Reply:
Sounds good 👍